controller/utils/utils.go

```diff
@@ -39,6 +39,7 @@ func JobConfig(f *ct.ExpandedFormation, name, hostID string, uuid string) *host.
 	metadata["flynn-controller.app"] = f.App.ID
 	metadata["flynn-controller.app_name"] = f.App.Name
 	metadata["flynn-controller.release"] = f.Release.ID
+	metadata["flynn-controller.formation"] = f.App.ID + ":" + f.Release.ID
```

Comment:
`JobConfig` takes a formation argument, so it's always used when starting a job for a formation (there are currently three uses: [scheduler](https://github.com/flynn/flynn/blob/v20160423.0/controller/scheduler/scheduler.go#L1335-L1337), [bootstrapper](https://github.com/flynn/flynn/blob/v20160423.0/bootstrap/run_app_action.go#L109-L120) and [host/fixer](https://github.com/flynn/flynn/blob/v20160423.0/host/fixer/fixer.go#L335-L343)), but the controller `RunJob` method [also sets the AppID and ReleaseID](https://github.com/flynn/flynn/blob/master/controller/jobs.go#L286-L288) so that they are associated with the app and release, but these are not formation jobs.

So whilst the actual formation ID of `AppID:ReleaseID` is redundant, the extra information serves to differentiate one-off jobs from formation jobs. We could just make this `flynn-controller.formation = true` if you prefer?